Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Model/ModelLod.h

```diff
@@ -81,7 +83,8 @@ namespace AZ
                 Data::Instance<Material> m_material;
             };
 
-            using StreamBufferViewList = AZStd::fixed_vector<RHI::SingleDeviceStreamBufferView, RHI::Limits::Pipeline::StreamCountMax>;
+            using StreamBufferViewList = AZStd::fixed_vector<RHI::MultiDeviceStreamBufferView, RHI::Limits::Pipeline::StreamCountMax>;
+            using TempStreamBufferViewList = AZStd::fixed_vector<RHI::SingleDeviceStreamBufferView, RHI::Limits::Pipeline::StreamCountMax>;
```

Comment:
Recommend using names
StreamBufferViewList = MultiDeviceStreamBufferViewList
and
TempStreamBufferViewList = SingleDeviceStreamBufferViewList

Reply:
This will be removed in the final rewrite and is only temporarily necessary.